config/project/fields/contentBlocksFaculty--aab7a3de-9a4a-4020-97b0-d960292b931f.yaml

```diff
@@ -0,0 +1,36 @@
+columnSuffix: null
+handle: contentBlocksFaculty
```

Comment:
Let’s stick to a common prefix for entry type as well as field handles dealing with the page builder blocks.